handlers/grafana.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/kiali/kiali/status"
```

Comment:
put this import in the proper place according to our coding standards (its a kiali import, so its in the bottom import group).

Reply:
don't forget to fix this - I don't see it fixed yet in the PR